src/geo/ui/widgets/time-series/torque-histogram-view.js

```diff
@@ -80,6 +80,7 @@ module.exports = View.extend({
 
   _onBrushEnd: function(loBarIndex, hiBarIndex) {
     var data = this.model.getData();
+    this._rangeFilter.set('columnType', this._torqueLayerModel.get('meta').column_type);
```

Comment:
As I pointed in the commit itself, I'm not happy with this approach. Any help to improve this is welcome.